README.md

```diff
@@ -339,6 +339,10 @@ Style to apply to the pager view wrapping all the scenes.
 
 Style to apply to the tab view container.
 
+### `animationEnabled`
```

Comment:
The heading level isn't right, move this to next to `swipeEnabled` and adjust the heading level to match it (`#####`)